api/controller.py

```diff
@@ -757,18 +757,29 @@ def _crawlable_feed(self, library, title, url, lane, annotator=None):
         )
         return feed_response(feed)
 
+    def _load_search_facets(self, lane):
+        entrypoints = list(flask.request.library.entrypoints)
+        if len(entrypoints) > 1:
+            # There is more than one enabled EntryPoint.
+            # By default, search them all.
+            default_entrypoint = EverythingEntryPoint
```

Comment:
What if there are multiple enabled entry points, but EverythingEntryPoint isn't one of them?

Reply:
EverythingEntryPoint is still the right choice. When you do a search, and there is more than one entry point, and you don't specify an entry point, you're searching everything, even if EverythingEntryPoint isn't an option for the site as a whole.

Selecting EverythingEntryPoint from the admin interface makes "All" show up as an option for navigation. It's always there for search, as long as there are multiple entry points enabled for navigation.